package main.Java.coreJavaPractice.gcr_codebase.String.LEVEL1;

public class NullPointerExceptionDemo {

    public static void handleException() {

        String text = null;

        try {
            System.out.println(text.length());
        } catch (NullPointerException e) {
            System.out.println("NullPointerException Handled");
        }
    }

    public static void main(String[] args) {

        handleException();
    }
}